### DISCUSSION TITLE
Manage elements style

### DISCUSSION THREAD

**Discussion by madibai** [iTwin/itwinjs-core#3557] [Q&A] [answered]
Hi
How i can hide or change element property?
Example change element color by switching ToggleSwitch click ?
Any tutorial or example ?
Thx

**Comment by pmconne** [comment]
[This sample](https://www.itwinjs.org/sandboxes/iTwinPlatform/Emphasize%20Elements) demonstrates how to hide elements and change their colors.
If that doesn't help please be more specific.

**Reply by madibai** [reply]
<img width="723" alt="Снимок экрана 2022-04-28 в 13 59 11" src="https://user-images.githubusercontent.com/42195915/165705801-81819742-8d62-4262-9098-bc20de6ba10e.png">

how replace sandbox functions?

**Reply by pmconne** [reply]
[Follow a tutorial](https://www.itwinjs.org/getting-started/).

**Reply by madibai** [reply]
I want manage color by ID of elements
For example get element ID from ecsql query or API and hide/show element by switching Toggle, so how can i get access to elements by ID ?

**Reply by pmconne** [reply]
You can for example use [IModelCollection.Elements.queryIds](https://www.itwinjs.org/reference/core-frontend/imodelconnection/imodelconnection/imodelconnection.elements/), then pass the element Ids to [EmphasizeElements.overrideElements](https://www.itwinjs.org/reference/core-frontend/rendering/emphasizeelements/overrideelements/).

**Reply by madibai** [reply]
How find element "V-301" ids ?

<img width="981" alt="Снимок экрана 2022-04-29 в 01 42 46" src="https://user-images.githubusercontent.com/42195915/165833135-f4c23181-ad8d-4aaf-9c53-b148b765783f.png">

overrideElements(ids ) ?

**Reply by pmconne** [reply]
Moved comment because github paging is quite bad.

**Comment by pmconne** [comment, marked as answer]
> How find element "V-301" ids ?
> 
> <img alt="Снимок экрана 2022-04-29 в 01 42 46" width="981" src="https://user-images.githubusercontent.com/42195915/165833135-f4c23181-ad8d-4aaf-9c53-b148b765783f.png">
> 
> overrideElements(ids ) ?

"V-301" is the element's [Code](https://www.itwinjs.org/reference/core-common/codes/code/). Try something like:
```
  const ids = await viewport.iModel.elements.queryIds({
    from: "BisCore.GeometricElement3d",
    where: "CodeValue='V-301'",
  });
  EmphasizeElements.getOrCreate(viewport).overrideElements(ids, viewport, ColorDef.green);
```

**Reply by madibai** [reply]
thx !!!
